include/circt/Dialect/Verif/Passes.td

```diff
@@ -15,40 +15,30 @@
 
 include "mlir/Pass/PassBase.td"
 
-/**
- * Performs preparation transformations for formal verification.
- * This includes things like flattening wires, removing unnecessary information
- * such as OM Classes, handles formal contracts and transforms module instances.
- */
- def PrepareForFormal : Pass<"prepare-for-formal", "hw::HWModuleOp"> {
-  let summary = "Prepares a given top-level circuit for formal verification.";
+def PrepareForFormalPass : Pass<"prepare-for-formal", "hw::HWModuleOp"> {
```

Comment:
nit: MLIR/CIRCT don't usually give "Pass" suffix to tablegen definition so it would be more consistent to keep that.

Reply:
The problem is that the constructors are then called `createPrepareForFormal`, which can be really strange if you have something like a "partition pass", which would create a constructor `createPartition`.